test/powershell/Modules/Microsoft.PowerShell.Core/Pester.Commands.Cmdlets.GetCommand.Tests.ps1

```diff
@@ -49,4 +49,52 @@ Describe "Tests Get-Command with relative paths and wildcards" -Tag "CI" {
         Pop-Location
     }
 
+    It "Get-Command -ShowCommandInfo property field test" {
+        $commandInfo = Get-Command Get-Date -ShowCommandInfo
```

Comment:
This seems to be doing `Get-Command Get-Date -ShowCommandInfo` a lot. This should be done once in `BeforeAll` and then used everywhere.

Reply:
resolved